src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -168,6 +148,11 @@ class AddressValidationModal extends React.Component {
     const showEditLink = showEditLinkErrorState || showEditLinkNonErrorState;
     const isFirstOptionOrEnabled =
       (isAddressFromUser && validationKey) || !isAddressFromUser;
+
+    const { street, cityStateZip, country } = formatAddress({
+      ...address,
```

Comment:
we could just pass the `address` directly without the spread but I know some people prefer this "just in case" the function would mess with the passed-in arg.

Reply:
I'll make the change to just pass in the address without the spread :)